scripts/install/conda-packages-list.txt

```diff
@@ -1,3 +1,33 @@
-pandas
+astroplan
+astropy >= 3.0.0
+ccdproc
```

Comment:
Even though conda fixes the problems with this (via astroscrappy), we are not actually using it anywhere in the repo. Might as well just remove.

Reply:
Remove ccdproc?

Reply:
I think so. It will come through in PIAA probably anyway but might as well get rid of it here since it's not an actual dependency.